src/battle_ai_util.c

```diff
@@ -21,6 +21,16 @@
 #include "constants/moves.h"
 #include "constants/items.h"
 
+#define CHECK_MOVE_FLAG(flag)                                                                                   \
```

Comment:
Wouldn't it be better to make it a function instead of a define?

Reply:
I don't know how to pass through references to the different members of a struct instead of their values, so I thought a define would suffice.

Reply:
I see what you mean, you can't pass a pointer to a bitfield struct member.

Reply:
@mrgriffin proposed a way but even he said that it was a monstrosity and shouldn't be used 😂